doc/zephyr.doxyfile.in

```diff
@@ -2366,6 +2366,8 @@ PREDEFINED             = __DOXYGEN__ \
                          CONFIG_FPU_SHARING \
                          CONFIG_GDBSTUB \
                          CONFIG_HEAP_MEM_POOL_SIZE \
+                         CONFIG_INPUT_KBD_ACTUAL_KEY_MASK_DYNAMIC \
+                         CONFIG_INPUT_KBD_DRIVE_COLUMN_HOOK \
```

Comment:
use `__DOXYGEN__` in code instead, you can find multiple examples in tree

Reply:
Thanks, forgot about that option somehow. Done.